backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/picking/PickingCandidate.java

```diff
@@ -66,7 +66,10 @@
 	private PickingCandidateApprovalStatus approvalStatus;
 
 	@NonNull
-	private final PickFrom pickFrom;
+	private PickFrom pickFrom;
+
+	@Nullable
+	private PickedFrom pickedFrom;
```

Comment:
why do we need PickFrom and PickedFrom?

Reply:
also in PickedFrom we have the Qty..... which shall be the PickingCandidate's qty....